README.md

```diff
@@ -65,6 +65,16 @@ And then execute:
 $ bundle
 ```
 
+## Semantic Versioning
+
+This project conforms to [semver](http://semver.org/). As a result of this policy, you can (and should) specify a dependency on this gem using the [Pessimistic Version Constraint](http://guides.rubygems.org/patterns/) with two digits of precision. For example:
+
+spec.add_dependency 'active_model_serializers', '~> 1.0'
+
+This means your project is compatible with licensee 1.0 up until 2.0. You can also set a higher minimum version:
```

Comment:
with version.  and maybe something like


```plain
[![SemVer 2.0](https://img.shields.io/badge/SemVer-2.0-lightgrey.svg)](http://semver.org/)
```

[![SemVer 2.0](https://img.shields.io/badge/SemVer-2.0-lightgrey.svg)](http://semver.org/)